src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.spec.ts

```diff
@@ -530,6 +530,28 @@ x4Ea7kGVgx9kWh5XjWz9wjZvY49UKIT5ppIAWPMbLl3UpfckiuNhTA==
       });
     });
 
+    describe('should test service mds', () => {
+      beforeEach(() => {
+        formHelper.setValue('service_type', 'mds');
+      });
+
+      it('should test mds valid service id', () => {
+        formHelper.setValue('service_id', 'svc123');
+        formHelper.expectValid('service_id');
+        formHelper.setValue('service_id', 'svc_id-1')
+        formHelper.expectValid('service_id');
+      });
+
+      it('should test mds invalid service id', () => {
+        formHelper.setValue('service_id', '123');
+        formHelper.expectError('service_id', 'mdsPattern');
+        formHelper.setValue('service_id', '123svc');
+        formHelper.expectError('service_id', 'mdsPattern');
+        formHelper.setValue('service_id', 'svc#1')
```

Comment:
@melissa-kun-li Make check is falling 
![image](https://user-images.githubusercontent.com/106810838/179909626-db7680be-43d4-4d70-8f87-1953166b2181.png)

You can see that it is warning you of some prettier issue with `[lint:prettier]`

In this very case it could be fixed with `npm run fix` since the issue is just this two missing semicolons

```suggestion
        formHelper.setValue('service_id', 'svc_id-1');
        formHelper.expectValid('service_id');
      });

      it('should test mds invalid service id', () => {
        formHelper.setValue('service_id', '123');
        formHelper.expectError('service_id', 'mdsPattern');
        formHelper.setValue('service_id', '123svc');
        formHelper.expectError('service_id', 'mdsPattern');
        formHelper.setValue('service_id', 'svc#1');
```